charmcraft.yaml

```diff
@@ -0,0 +1,27 @@
+type: charm
+bases:
+- build-on:
+  - name: "ubuntu"
+    channel: "20.04"
+  run-on:
+  - name: "ubuntu"
+    channel: "20.04"
+    architectures: [amd64, arm64]
+- build-on:
+  - name: "ubuntu"
+    channel: "22.04"
+  run-on:
+  - name: "ubuntu"
+    channel: "22.04"
+    architectures: [amd64, arm64]
+parts:
+  charm:
+    source: .
+    charm-requirements: [requirements.txt]
```

Comment:
I couldn't find anything in the docs to use a `pyproject.toml` file instead. Am I correct in understanding that we have no choice but to use a `requirements.txt` file for this still in 2023? :(

Reply:
For some context, this is plugin for the `parts` section in the charmcraft.yaml file. There are other plugins like `bundle` as well. The charm plugin provides the key for `charm-requirements` or `charm-python-packages` the first one takes as the description for it [says](https://juju.is/docs/sdk/charmcraft-yaml#heading--parts), a list of requirements files. So I don't think they support the pyproject.toml format.

Reply:
Too bad. Thanks for the explanation!